sdk/cosmos/azure-cosmos/azure/cosmos/database.py

```diff
@@ -159,6 +167,12 @@ def create_container(  # pylint:disable=docstring-missing-param
         offer_throughput: Optional[Union[int, ThroughputProperties]] = None,
         unique_key_policy: Optional[Dict[str, Any]] = None,
         conflict_resolution_policy: Optional[Dict[str, Any]] = None,
+        *,
+        session_token: Optional[str] = None,
+        initial_headers: Optional[Dict[str, str]] = None,
+        etag: Optional[str] = None,
+        match_condition: Optional[MatchConditions] = None,
+        analytical_storage_ttl: Optional[int] = None,
```

Comment:
We're not adding `computed_properties` here yet?
I think I also recall you mentioning that this was still a preview feature? Thought it does not appear to be documented as such...

Reply:
that's a good point, I will go ahead and update the docs to make it show up as provisional. I only didn't elevate it because I saw it wasn't elevated in the async client and assumed the reasoning was due to it being in preview - I can go ahead and update all of it if that's not the case and it was just missed :)